Packs/1_1_3SuperchargingXSOARWithAnsible/Integrations/Linux/Linux.yml

```
@@ -0,0 +1,6753 @@
+    - default: true
+      defaultValue: ansible.
+      description: Prefix of file/directory name created by module.
+      name: prefix
+    - default: true
+      description: Suffix of file/directory name created by module.
+      name: suffix
+    description: Creates temporary files and directories
+    name: linux_tempfile
+  - arguments:
+    - description: hostname or IP of target. Optionally the port can be specified
+        using :PORT. If multiple targets are specified using an array, the integration
+        will use the configured concurrency factor for high performance.
+      isArray: true
+      name: host
+      required: true
+    - description: |-
+        If `remote_src=no` (default), local path to archive file to copy to the target server; can be absolute or relative. If `remote_src=yes`, path on the target server to existing archive file to unpack.
+        If `remote_src=yes` and `src` contains `://`, the remote machine will download the file from the URL first. (version_added 2.0). This is only for simple cases, for full download support use the `get_url` module.
+      name: src
+      required: true
+    - description: Remote absolute path where the archive should be unpacked.
+      name: dest
+      required: true
+    - default: true
+      defaultValue: "True"
+      description: |-
+        If true, the file is copied from local 'master' to the target machine, otherwise, the plugin will look for src archive at the target machine.
+        This option has been deprecated in favor of `remote_src`.
+        This option is mutually exclusive with `remote_src`.
+      name: copy
+    - description: If the specified absolute path (file or directory) already exists,
+        this step will `not` be run.
+      name: creates
+    - default: true
+      defaultValue: "False"
+      description: If set to True, return the list of files that are contained in
+        the tarball.
+      name: list_files
+    - description: List the directory and file entries that you would like to exclude
+        from the unarchive action.
+      isArray: true
+      name: exclude
+    - default: true
+      defaultValue: "False"
+      description: Do not replace existing files that are newer than files from the
+        archive.
+      name: keep_newer
+    - default: true
+      description: |-
+        Specify additional options by passing in an array.
+        Each space-separated command-line option should be a new element of the array. See examples.
+        Command-line options with multiple elements must use multiple lines in the array, one for each element.
+      isArray: true
+      name: extra_opts
+    - default: true
+      defaultValue: "False"
+      description: |-
+        Set to `yes` to indicate the archived file is already on the remote system and not local to the Ansible controller.
+        This option is mutually exclusive with `copy`.
+      name: remote_src
+    - default: true
+      defaultValue: "True"
```

```suggestion
      defaultValue: "False"
```
It would be better that validate certs would be set to False and only if user would like he will set it to true.

---

Thank you for the feedback. I have used the Ansible default value here as per https://docs.ansible.com/ansible/2.9/modules/copy_module.html

I suggest that as per the design philosophy of "Secure by Default" certificates should be validated by default, so I agree with the default value used in the Ansible module in this case.